crates/holochain/src/core/state/cascade.rs

```diff
@@ -340,7 +341,12 @@ where
             }
             Activity::NotRequested => (),
         };
-        cache_data.meta.register_activity_status(&agent, status)?;
+        match &status {
+            ChainStatus::Empty => {}
+            ChainStatus::Valid(_) | ChainStatus::Forked(_) | ChainStatus::Invalid(_) => {
+                cache_data.meta.register_activity_status(&agent, status)?;
+            }
+        }
```

Comment:
Because we can't add empty statuses to the metadata. Empty must be represented by the absence of a status otherwise we would need to enter an empty status for all chain possibilities which is impossible. Empty is just for sending across the network that we don't have any information.